tests/virt_autotest/libvirt_nated_virtual_network.pm

```diff
@@ -40,34 +40,28 @@ sub run_test {
     die "The default(NAT BASED NETWORK) virtual network does not exist" if (script_run('virsh net-list --all | grep default') != 0);
 
     #Create NAT BASED NETWORK
-    assert_script_run("virsh net-info default");
-    assert_script_run("virsh net-dumpxml default |tee libvirt_default.xml");
-    assert_script_run("virsh net-undefine default");
+    #assert_script_run("virsh net-info default");
```

Comment:
Before start or create NAT BASED Network, please destroy default virtual network - default nat mode firstly. We need as clean a test environment as possible.

Reply:
This is not needed - I bind my test HTTP request from the IP we got from the new network and I route it via this network.

Reply:
Okay, got it.